### PR TITLE
`pyupgrade` the code

### PR DESCRIPTION
**Pull request by pawelrubin**
#### Codebase upgrade via `pyupgrade --py36-plus`.
Since the **aiokafka** package is not compatible with Python versions lower than 3.6, the codebase can be upgraded.

### Checklist

I believe non of the following is applicable.

- [ ] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` (e.g. `588.bugfix`)
  * if you don't have an `issue_id` change it to the pr id after creating the PR
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: `Fix issue with non-ascii contents in doctest text files.`

### CODE REVIEW

File: aiokafka/conn.py [hunk 1]
```diff
@@ -414,7 +414,7 @@ def port(self):
     def send(self, request, expect_response=True):
         if self._writer is None:
             raise Errors.KafkaConnectionError(
-                "No connection to broker at {0}:{1}"
+                "No connection to broker at {}:{}"
```
Comment (ods) [inline aiokafka/conn.py:417]: `{self._host}:{self._port}`? And in other similar places below

**Review by ods — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 2]
```diff
@@ -72,3 +72,9 @@ aiokafka/record/_crecords/legacy_records.c
 aiokafka/record/_crecords/memory_records.c
 aiokafka/record/_crecords/cutil.c
 aiokafka/record/_crecords/*.html
+
+# VS Code
+.vscode
```
[the remark this answers is not in the document]
Comment (pawelrubin) [reply .gitignore]: I honestly didn't know there is such a possibility. 👀 
I've dropped the commit.

File: aiokafka/conn.py (hunk 1 above)
Comment (pawelrubin) [reply aiokafka/conn.py]: Sure thing. Unfortunately, **pyupgrade** cannot `pyupgrade` multiline strings, so I had to do it half-manually with a regex.

FYI, related [issue](https://github.com/asottile/pyupgrade/issues/55) in the **pyupgrade** repo

File: aiokafka/conn.py [hunk 3]
```diff
@@ -443,7 +441,7 @@ def send(self, request, expect_response=True):
     def _send_sasl_token(self, payload, expect_response=True):
         if self._writer is None:
             raise Errors.KafkaConnectionError(
-                "No connection to broker at {0}:{1}"
+                "No connection to broker at {}:{}"
                 .format(self._host, self._port))
```
Comment (ods) [inline aiokafka/conn.py:445]: ?

**Review by ods — CHANGES_REQUESTED** [review]

File: tests/test_coordinator.py [hunk 4]
```diff
@@ -73,14 +73,14 @@ async def test_coordinator_workflow(self):
         self.assertFalse(coordinator.need_rejoin(subscription.subscription))
 
         tp_list = subscription.assigned_partitions()
-        self.assertEqual(tp_list, set([('topic1', 0), ('topic1', 1),
-                                       ('topic2', 0), ('topic2', 1)]))
+        self.assertEqual(tp_list, {('topic1', 0), ('topic1', 1),
+                                       ('topic2', 0), ('topic2', 1)})
```
Comment (ods) [inline tests/test_coordinator.py:77]: https://github.com/aio-libs/aiokafka/pull/752/checks?check_run_id=2832624476#step:9:12
You can run `make lint` locally to see all linting errors

File: aiokafka/conn.py (hunk 3 above)
Comment (pawelrubin) [reply aiokafka/conn.py]: I've missed this, and some other `format` occurrences. Fixed in c7e78f19bf2dea2b3a10fd82273cc3b0f7144e5c

File: tests/test_coordinator.py (hunk 4 above)
Comment (pawelrubin) [reply tests/test_coordinator.py]: Fixed in 642fcb13eb0ff2c59665ff117b66f84c167f621b

**Pull request merged**